### ISSUE TITLE
Error -5200, Failed to create FullscreenVideoAdPresenter

### ISSUE THREAD

**Issue by mmokona** [AppLovin/AppLovin-MAX-SDK-Android#36]
``` 
2021-05-19 18:51:14.211 23230-23230/? E/AppLovinSdk: [InterActivityV2] Failed to create FullscreenVideoAdPresenter with sdk: CoreSdk{sdkKey='<hidden>', enabled=true, isFirstSession=true} and throwable: Class com.applovin.impl.adview.activity.a$d extended by class com.applovin.impl.adview.activity.a.c is inaccessible (declaration of 'com.applovin.impl.adview.activity.a.c' appears in base.apk)
    java.lang.IllegalAccessError: Class com.applovin.impl.adview.activity.a$d extended by class com.applovin.impl.adview.activity.a.c is inaccessible (declaration of 'com.applovin.impl.adview.activity.a.c' appears in base.apk)
        at com.applovin.adview.AppLovinFullscreenActivity.present(Unknown Source:267)
        at com.applovin.adview.AppLovinFullscreenActivity.onCreate(Unknown Source:144)
        at android.app.Activity.performCreate(Activity.java:7822)
        at android.app.Activity.performCreate(Activity.java:7811)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1328)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3430)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3598)
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:83)
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2164)
        at android.os.Handler.dispatchMessage(Handler.java:107)
        at android.os.Looper.loop(Looper.java:241)
        at android.app.ActivityThread.main(ActivityThread.java:7582)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:941)
2021-05-19 18:51:14.211 23230-23230/? W/AppLovinSdk: [MediationAdapterWrapper] AppLovinMediationAdapter: rewarded ad display failed with error: MaxAdapterError{errorCode=-5200, errorMessage="Failed to create FullscreenVideoAdPresenter with sdk: CoreSdk{sdkKey='<hidden>', enabled=true, isFirstSession=true} and throwable: Class com.applovin.impl.adview.activity.a$d extended by class com.applovin.impl.adview.activity.a.c is inaccessible (declaration of 'com.applovin.impl.adview.activity.a.c' appears in base.apk)", thirdPartySdkErrorCode=0, thirdPartySdkErrorMessage=""}  
```
Sdk: v10.2.1
Android: v10
This error occurs when calling Interstitial and Rewarded video, and only in the release build. Debug build work fine. Any else mediation sdks work fine.

**Comment by Lorenzo45** [comment]
Are you using our Unity Plugin as well? If so, which version?
Are you making an AAB for the release build or an APK?

**Comment by mmokona** [comment]
1) No, its android gradle integration.
2) APK, signature v1 and v2

**Comment by Lorenzo45** [comment]
Thanks for the info, we'll try to reproduce the crash on our end. If possible, would you be able to send your Android project to my email lorenzo.gentile@applovin.com? That would be super helpful. Otherwise, any info about your project would be helpful. Specifically, the contents of your build.gradle file or any proguard rules you have. Thanks!

**Comment by thomasmso** [comment]
bump @mmokona

**Comment by thomasmso** [comment]
@mmokona - This should be fixed in the latest SDK release v11.0.0.
